MSAL/src/public/MSALResult.h

```diff
@@ -100,4 +100,8 @@
  */
 @property (readonly, nonnull) NSUUID *correlationId;
 
+- (NSString *_Nonnull)getAuthorizationHeader;
```

Comment:
I think this should be a readonly property instead

Reply:
I think this is just a method not a property so we should be good. I think it is confusing because Objc creates getters in the same format.

Reply:
Addressed. Converted it to readonly property instead.